client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -88,19 +85,18 @@ export function getEnhancedTasks(
 				case 'links_added':
 					taskData = {
 						title: translate( 'Add links' ),
+						completed: linkInBioLinksEditCompleted,
 						actionDispatch: () => {
-							recordTaskClickTracksEvent( flow, task.isCompleted, task.id );
+							recordTaskClickTracksEvent( flow, linkInBioLinksEditCompleted, task.id );
 							window.location.replace( `/site-editor/${ siteSlug }` );
 						},
-						keepActive: true,
-						isCompleted: linkInBioLinksEditCompleted,
 					};
 					break;
 				case 'link_in_bio_launched':
 					taskData = {
 						title: translate( 'Launch Link in bio' ),
-						isCompleted: linkInBioSiteLaunchCompleted,
-						dependencies: [ linkInBioLinksEditCompleted ],
+						completed: linkInBioSiteLaunchCompleted,
+						disabled: ! linkInBioLinksEditCompleted,
```

Comment:
simplify the dependencies implementation by modifying the `disabled` prop directly